src/components/Navbar/Account.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+interface NavbarAccount {
+  isUserExist?: boolean
+}
+
+const NavbarAccount = ({ isUserExist }: NavbarAccount) => {
+  const router = useRouter()
+
+  const handleSignInClick = () => {
+    // TODO: Open SignIn modal
+  }
+
+  const handleSignUpClick = () => {
+    router.push('/sign-up')
+  }
+
+  const handleAvatarClick = () => {
+    // TODO: Expand dropdown or sth
+  }
+
+  return (
+    <div className="navbar__account">
+      {isUserExist ? (
+        <button className="navbar__avatar-button" onClick={handleAvatarClick}>
+          <img src="#" alt="avatar" />
+        </button>
+      ) : (
+        <>
+          <button
+            className="navbar__account-button"
+            onClick={handleSignInClick}>
+            Sign In
+          </button>
+          <button
+            className="navbar__account-button"
+            onClick={handleSignUpClick}>
+            Sign Up
+          </button>
+        </>
+      )}
+    </div>
```

Comment:
What do you think of this? Create a variable for those elements for better readability. 🤔
```suggestion
    <div className="navbar__account">
      {isUserExist ? (navbarAvatar) : (navbarAuthentication)}
    </div>
```

Reply:
Sure, it looks a lot cleaner. But I would follow how Airbnb names their methods that return JSX partials.
Ref: https://github.com/airbnb/react-dates/blob/master/src/components/DateRangePicker.jsx#L317

Reply:
According to your reference, I can only see that they don't even include conditional rendering at `render()` rather they put it on the functions instead. Furthermore, on `return()` they even put a component there instead of the components body there. In addition, from React's [docs](https://reactjs.org/docs/conditional-rendering.html), you can see at the render/return they keep it in one **LOC**. I don't think it's a good idea to use conditional parameter in your way. Does my opinion changes yours?

Reply:
What I followed from `Airbnb` is how they name there methods not how they use component's styling.
This is what I adapt.
<img width="256" alt="Screen Shot 2564-02-02 at 16 20 34" src="https://user-images.githubusercontent.com/32285706/106578864-96772580-6572-11eb-97d4-12ed849c86d6.png">